evalml/pipelines/components/estimators/regressors/varmax_regressor.py

```diff
@@ -217,9 +217,43 @@ def get_prediction_intervals(
         Returns:
             dict: Prediction intervals, keys are in the format {coverage}_lower or {coverage}_upper.
         """
-        raise NotImplementedError(
-            "VARMAX does not have prediction intervals implemented yet.",
+        if coverage is None:
+            coverage = [0.95]
+
+        X, y = self._manage_woodwork(X, y)
+        if self.use_covariates:
+            X = X.ww.select(exclude=["Datetime"])
+            X = convert_bool_to_double(X)
+        # Accesses the fitted statsmodels model within sktime
+        # nsimulations represents how many steps should be simulated
+        # repetitions represents the number of simulations that should be run (confusing, I know)
+        # anchor represents where the simulations should start from (forecasting is done from the "end")
+        y_pred = self._component_obj._fitted_forecaster.simulate(
+            nsimulations=X.shape[0],
+            repetitions=400,
```

Comment:
Hmm, poking around in our exponential smoother and statsmodels' docs on the subject, it's unclear to me why this was set at 400. I think at least setting it as a constant would be good, since the number seems arbitrary.

Reply:
Sounds good, will update to include `_N_SIMULATIONS=400`